dbf2csv/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 INGETIVE (<https://ingetive.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Convert DBF to CSV',
+    'version': '10.0.1.0.0',
+    'author': 'Félix Villafranca (Ingetive),'
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.ingetive.com',
+    'license': 'AGPL-3',
+    'category': 'Usability',
+    'contributors': [
+        'Félix Villafranca <f.villafranca@ingetive.com>',
+    ],
+    'summary': "Convert DBF file to CSV file",
+    'depends': [
+        'base',
+    ],
+    'data': [
+        'views/views.xml',
+    ],
+    'external_dependencies': {
+        'python': ['dbfpy']
```

Comment:
From [OCA idioms](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms):

> When making a comma separated list, dict, tuple, ... with one element per line, append a comma to the last element. This makes it so the next element added only changes one line in the changeset instead of changing the last element to simply add a comma.

Reply:
Ok, thanks